src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -399,6 +401,14 @@ def set_feedback(feedback):
     _session.feedback = feedback[:512]
 
 
+# This function returns the user's selection and feedback on the cli-recommendation results
+# Please refer to feedback_design.md of cli-recommendation for detailed information
+@decorators.suppress_all_exceptions()
+def set_cli_recommendation(api_version, cli_recommendation_feedback):
+    _session.cli_recommendation = {"api_version": api_version,
+                                   "cli_recommendation_feedback": cli_recommendation_feedback}
```

Comment:
```suggestion
@decorators.suppress_all_exceptions()
def set_cli_recommendation(api_version, feedback):
    # This function returns the user's selection and feedback on the cli-recommendation results
    # Please refer to feedback_design.md of cli-recommendation for detailed information
    _session.cli_recommendation = {"api_version": api_version,
                                   "feedback": feedback}
```

Reply:
Received, already modified and also related update in az interactive https://github.com/wu5bocheng/azure-cli-extensions/commit/ef2dfbb64f4bf20ebe3d67fc7936e16262f0e09b